kernel/init.c

```diff
@@ -51,7 +51,7 @@ const char * const build_timestamp = BUILD_TIMESTAMP;
 /* boot time measurement items */
 
 #ifdef CONFIG_BOOT_TIME_MEASUREMENT
-u64_t __noinit __start_time_stamp; /* timestamp when kernel starts */
+u64_t __start_time_stamp; /* timestamp when kernel starts */
```

Comment:
Ok, wasn't thinking about when bss gets cleared vs this gets set on x86.  Will move back to __noinit and explicitly setting to 0 on ARM.